### PR TITLE
CV2-5048: Delete the user profile page

### PR DESCRIPTION
**Pull request by danielevalverde**
## Description

- [X] Delete user route
- [X] Delete the <User /> component 
- [X] Remove any links to the user profile
- [X] Delete orphaned code, as reported by ESLint
- [X]  Review if, after deleting all the code above, there is still any code that requests image from user.source - if so, it should request profile_image from user, instead

Reference: CV2-5048

## Type of change

- [X] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/task/Task.js [hunk 1]
```diff
@@ -531,13 +529,7 @@ class Task extends Component {
...
         responseObj && responseObj.annotator ? (
           <div className={styles['task-footer']}>
-            Saved {timeAgo} by{' '}
-            <a
-              href={`/check/user/${responseObj.annotator.user?.dbid}`}
-              title={responseObj.annotator.user?.name}
-            >
-              {responseObj.annotator.user?.name}
-            </a>
+            {`Saved ${timeAgo} by ${responseObj.annotator.user?.name}`}
```
Comment (amoedoamorim) [inline src/app/components/task/Task.js:532]: This string must be translatable

File: src/app/components/task/Tasks.js [hunk 2]
```diff
@@ -222,12 +222,7 @@ const Tasks = ({
     }
     return (
       <div className={inputStyles['form-footer-actions-context']}>
-        Saved {moment(latestEditInfo.latestDate).fromNow()} by{' '}
-        <a
-          href={`/check/user/${latestEditInfo.latestAuthorDbid}`}
-        >
-          {latestEditInfo.latestAuthorName}
-        </a>
+        {`Saved ${moment(latestEditInfo.latestDate).fromNow()} by ${latestEditInfo.latestAuthorName}`}
```
Comment (amoedoamorim) [inline src/app/components/task/Tasks.js:225]: This string also must be translated. Ideally this has the same string id as the other instance.

File: src/app/components/task/Task.js (hunk 1 above)
Comment (danielevalverde) [reply src/app/components/task/Task.js]: done :white_check_mark:

File: src/app/components/task/Tasks.js (hunk 2 above)
Comment (danielevalverde) [reply src/app/components/task/Tasks.js]: done :white_check_mark:

**Comment by danielevalverde** [comment]
Thanks @amoedoamorim , applied the PR review :)

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**